### PR TITLE
Yandex update

### LINKED ISSUE #3317: Yandex Metrica to setup

### LINKED ISSUE #3321: Update Yandex Metrica setup

### PR DESCRIPTION
**Pull request by Mussuranae**
Update Yandex metrica and id for every app
Fix #3317 and fix #3321

### CODE REVIEW

File: apps/catalog/catalog/src/app/app.module.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ import { sentryDsn } from '@env';
 
 // Yandex Metrika
 import { YandexMetricaModule } from '@blockframes/utils/yandex-metrica/yandex-metrica.module'
-import { yandexId } from '@env';
+import { catalogYandexId } from '@env';
```
Comment (GrandSchtroumpf) [inline apps/catalog/catalog/src/app/app.module.ts:35]: Prefer : `yandexId.catalog`. It's easier to update after.

File: env/demo/env.demo1.ts [hunk 2]
```diff
@@ -130,7 +130,8 @@ export const templateIds = {
 }
 
 // Yandex Metrika Id
-export const yandexId = 0;
+export const festivalYandexId = 0;
```
Comment (GrandSchtroumpf) [inline env/demo/env.demo1.ts:133]:
```typescript
export yandexId = {
  catalog: 0,
  festival: 0,
}

File: libs/utils/src/lib/yandex-metrica/yandex-metrica.service.ts [hunk 3]
```diff
@@ -10,34 +10,35 @@ export class YandexMetricaService {
...
+
+      // Append script tag into the head
+      const insetScriptTag = () => head.appendChild(script);
+      // Fix for Opera browser
+      if ((window as any).opera === '[object Opera]') {
+        this.document.addEventListener('DOMContentLoaded', insetScriptTag, false);
+      } else {
+        insetScriptTag();
+      }
     } else {
```
Comment (GrandSchtroumpf) [inline libs/utils/src/lib/yandex-metrica/yandex-metrica.service.ts:40]: You don't need that. This is the default behavior

**Comment by fritzschoff** [comment]
@Mussuranae merge?

**Pull request closed without merging**